src/Testcontainers.Pulsar/PulsarContainer.cs

```diff
@@ -63,9 +63,9 @@ public async Task<string> CreateAuthenticationTokenAsync(TimeSpan expiryTime, Ca
         {
             int secondsToMilliseconds;
 
-            if (_configuration.Image.Tag.StartsWith("3.2") || _configuration.Image.Tag.StartsWith("latest"))
+            if (_configuration.Image.Tag.Equals("3.2.0") || _configuration.Image.Tag.Equals("3.2.1") || _configuration.Image.Tag.Equals("3.2.2") || _configuration.Image.Tag.Equals("3.2.3") || _configuration.Image.Tag.Equals("3.3.0") )
```

Comment:
FYI: We now have a new [API](https://github.com/testcontainers/testcontainers-dotnet/blob/develop/tests/Testcontainers.Tests/Unit/Images/TestcontainersImageTest.cs#L111-L119) available to check the image version.

Reply:
Very nice! I'll adopt that!